### ISSUE TITLE
Custom Button Cell

### ISSUE THREAD

**Issue by albertml** [jerkoch/SwipeCellKit#454]
Is this achievable were the action buttons has spacing and round corners? See attached image.
![Screen Shot 2023-06-02 at 6 03 33 PM](https://github.com/SwipeCellKit/SwipeCellKit/assets/8196166/2ff78fa9-1742-4d21-93df-b2c9a7b712a1)

**Comment by GregOriol** [comment]
@albertml You could try to subclass `SwipeActionsView` and make your own custom layout there, instead of the simple buttons used in the implementation

**Comment by ovidiuadorian-yonder** [comment]
@GregOriol The library does not seem to support subclass of `SwipeActionsView `. Or you are thinking of a fork?

**Comment by GregOriol** [comment]
@ovidiuadorian-yonder Indeed, you can fork it and make `SwipeActionsView` public. The use-case described here is not really what the library is created for, it's aim is more to re-implement the standard iOS behavior, but it's base mechanisms can have many more custom ideas implemented on top of it.
